pkg/Library/App/Worker/App.js

```diff
@@ -49,11 +49,13 @@ export const App = class {
         return this.persist(request);
       case 'restore':
         return this.restore(request);
-      default: {
-        return await (this.dispatchServiceRequest({request}) || this.appServices({request}));
-      }
     }
-  }
+    const result =
+      this.dispatchServiceRequest()
```

Comment:
did you remove the `{request}` param from `this.dispatchServiceRequest` on purpose? it will never return anything at all in this case...